### DISCUSSION TITLE
Web server in asyncio (aiohttp)

### DISCUSSION THREAD

**Discussion by Jin-Xu** [EddieHubCommunity/support#190] [General] [answered]
Just wondering if anyone had any experience building/running a web server in aiohttp, is this reliable? I've been using it for a while but not necessarily in a commercial environment. Thanks.

**Comment by eddiejaoude** [comment]
I have shared this discussion in our [Discord](https://discord.com/invite/jZQs6Wu) Python channel, hopefully someone can share some experience on this 🤓

**Reply by Jin-Xu** [reply]
Cool, thanks.

**Comment by Jin-Xu** [comment, marked as answer]
This might be irrelevant as it would appear there are no clear benefits of using asyncio just for the sake of performance as it would appear that there is none http://calpaterson.com/async-python-is-not-faster.html
